deployment/data-streams/changeset/jd_register_nodes.go

```diff
@@ -80,17 +99,15 @@ func (cfg RegisterNodesInput) Validate() error {
 		return errors.New("ProductName must not be empty")
 	}
 
-	for donName, don := range cfg.DONs {
+	for i, don := range cfg.DONsList {
```

Comment:
nit: one other validation you can add is that a don must have bootstrap nodes

Reply:
Added it too!